microsoft-edge/devtools-guide-chromium/storage/cookies.md

```diff
@@ -58,43 +57,46 @@ The **Cookies** table contains the following fields:
 
 *  **Size**.  The size, in bytes, of the cookie.
 
-*  **HTTP**.  If true, this field indicates that the cookie should only be used over HTTP, and JavaScript modification isn't allowed.  See [HttpOnly cookies](https://developer.mozilla.org/docs/Web/HTTP/Cookies#Secure_and_HttpOnly_cookies).
+*  **HttpOnly**.  If true, this field indicates that the cookie should only be used over HTTP, and JavaScript modification isn't allowed.  See [HttpOnly cookies](https://developer.mozilla.org/docs/Web/HTTP/Cookies#Secure_and_HttpOnly_cookies).
```

Comment:
```suggestion
*  **HttpOnly**.  If `true`, this field indicates that the cookie should only be used over HTTP, and JavaScript modification isn't allowed.  See [HttpOnly cookies](https://developer.mozilla.org/docs/Web/HTTP/Cookies#Secure_and_HttpOnly_cookies).
```
if ui, use bold instead of backticks.  User-entered text in a ui is bolded. Editing code in a code file is backticks.